mmv1/third_party/terraform/tests/resource_compute_region_instance_group_manager_test.go.erb

```diff
@@ -1405,7 +1405,7 @@ resource "google_compute_region_instance_group_manager" "igm-basic" {
     device_name = "stateful-disk"
     delete_rule = "NEVER"
   }
-  <% unless version == "ga" -%>
```

Comment:
Could the changes to this file be unreverted? seems unrelated to the change.

Reply:
yes sorry, I changed it when debugging some weird VCR test failure.